pysat/_constellation.py

```diff
@@ -136,6 +277,62 @@ def bounds(self, value=None):
 
         return
 
+    @property
+    def date(self):
+        """Date for loaded data."""
+        if len(self.index) > 0:
+            return utils.time.filter_datetime_input(self.index[0])
+        else:
+            return None
+
+    @property
+    def empty(self):
+        """Boolean flag reflecting lack of data, True if there is no Instrument
+        data in any Constellation Instrument.
+        """
+        return self._empty(all_inst=True)
+
+    @property
+    def index(self):
+        """Returns time index of loaded data."""
+        return self._index()
+
+    def today(self):
+        """Returns UTC date for today, see pysat.Instrument for details."""
+        return utils.time.today()
```

Comment:
I'm guessing if you thought `tomorrow` and `yesterday` should also be implemented in `utils.time` you would have. I am curious, why `today` and not the others?

Reply:
For the written record, because `tomorrow` and `yesterday` are simple additions and `today` is the time call that actually does much work.  Also, grabbing `today` from Instrument didn't work (caused `pysat` to not import 🦇 ) and that motivated the move to `utils.time`.